test/unit/display_svg_spec.js

```diff
@@ -101,35 +108,35 @@ describe('SVGGraphics', function () {
       });
     }
 
-    it('should produce a reasonably small svg:image', function() {
+    it('should produce a reasonably small svg:image', function(done) {
       if (!isNodeJS()) {
         pending('zlib.deflateSync is not supported in non-Node environments.');
       }
       withZlib(true, getSVGImage).then(function(svgImg) {
         expect(svgImg.nodeName).toBe('svg:image');
-        expect(svgImg.getAttribute('width')).toBe('200px');
-        expect(svgImg.getAttribute('height')).toBe('100px');
-        var imgUrl = svgImg.getAttribute('xlink:href');
+        expect(svgImg.getAttributeNS(null, 'width')).toBe('200px');
+        expect(svgImg.getAttributeNS(null, 'height')).toBe('100px');
+        var imgUrl = svgImg.getAttributeNS(XLINK_NS, 'href');
         // forceDataSchema = true, so the generated URL should be a data:-URL.
         expect(imgUrl).toMatch(/^data:image\/png;base64,/);
         // Test whether the generated image has a reasonable file size.
         // I obtained a data URL of size 366 with Node 8.1.3 and zlib 1.2.11.
         // Without zlib (uncompressed), the size of the data URL was excessive
         // (80247).
```

Comment:
Nit: Shouldn't this be updated to read `80246` now, given the change at line https://github.com/mozilla/pdf.js/pull/8651/files#diff-5d0a387a1622d792d0bf53673e074238R138 below?